beacon/weakcoin/weak_coin.go

```diff
@@ -21,6 +21,9 @@ var (
 	errNotSmallest  = errors.New("proposal not smallest")
 )
 
+// biggest is 1 bigger than the largest value that can be represented by a VRF signature.
+var biggest = new(big.Int).Lsh(big.NewInt(1), types.VrfSignatureSize*8)
```

Comment:
why use a big.Int to do this? why not a byte array of 1s?

Reply:
that would be incorrect, see my other comment, but I'll try to make it work with arrays 🙂